lib/src/core/room.dart

```diff
@@ -165,11 +165,12 @@ class Room extends DisposableChangeNotifier with EventsEmittable<RoomEvent> {
     RoomOptions? roomOptions,
     FastConnectOptions? fastConnectOptions,
   }) {
-    if (roomOptions?.e2eeOptions != null) {
+    roomOptions ??= this.roomOptions;
```

Comment:
good fix.. in the future I think we should deprecate roomOptions from `connect`. this should really be set in Room constructor.